### PR TITLE
Fix gh-1444: Change Things to Try to Tips

### LINKED ISSUE #1444: Update Educator page to reflect new Tips page
It would be good to update the Educator page to refer to the Tips page.

1) Changing wording from "Things to Try" --> "Tips". 
2) Changing link from https://scratch.mit.edu/go -->  https://scratch.mit.edu/tips
3) Changing icon if possible: It would also be helpful to have a different icon for that page 
@carljbowman could a simplified version of the awesome cards icon from the Tips page work? 

Here is the current wording and icon:
image

### PR DESCRIPTION
**Pull request by St19Galla**
Fixes #1444 by changing the Things to Try text to Tips, and same with the link.

I fixed this as it had the help-wanted tag, but I did not create a new image. @carljbowman if you have the new image you can send it to me on Slack and I can add it in, or feel free to close this PR and open another :)

### CODE REVIEW

**Comment by St19Galla** [comment]
/cc @carljbowman @chrisgarrity 

I think this is complete and ready now :)

**Comment by carljbowman** [comment]
@St19Galla - Would you mind tossing up a quick screenshot. Thanks in advance!

File: src/views/teachers/landing/landing.jsx [hunk 1]
```diff
@@ -85,7 +85,7 @@ var Landing = injectIntl(React.createClass({
                         <FlexRow className="guides-and-tutorials">
                             <div>
                                 <a href="/go">
```
Comment (chrisgarrity) [inline src/views/teachers/landing/landing.jsx]: This href should also be `/tips`.

File: src/views/teachers/landing/landing.jsx [hunk 2]
```diff
@@ -85,7 +85,7 @@ var Landing = injectIntl(React.createClass({
...
-                                    <img src="/svgs/teachers/resources.svg" alt="resources icon" />
+                                    <img src="/svgs/teachers/cards.svg" alt="cards icon" />
```
Comment (chrisgarrity) [inline src/views/teachers/landing/landing.jsx]: The new image doesn't match the size/aspect ration of the others so the text isn't aligned. You'll need to get a new version of the SVG from @carljbowman

Comment (carljbowman) [inline src/views/teachers/landing/landing.jsx]: Sorry about that @St19Galla. Did not realize we had a consistent image ratio on this page. 

Here is a new SVG with the correct ratio: 
[v2-cards.svg.zip](https://github.com/LLK/scratch-www/files/1188513/v2-cards.svg.zip)

Comment (chrisgarrity) [inline src/views/teachers/landing/landing.jsx]: @St19Galla if you want to update the PR with the new image, I'll re-review. Thanks!

**Comment by St19Galla** [comment]
@chrisgarrity whoops! sorry this is so late, I've been super busy :/ Done!

**Comment by chrisgarrity** [comment]
Thanks @St19Galla, it looks great!

**Comment by Kenny2github** [comment]
Out of curiosity, Is this another PR that shouldn’t deploy before 7/8?

**Comment by chrisgarrity** [comment]
This one will probably just get deployed in our regular monthly schedule. Ones that have to do with job postings and team changes often get deployed as soon as they're ready because they're time critical.

**Comment by mewtaylor** [comment]
On Chrome, OSX 10.12
* new image appears
* the text reads "Tips page"
* the link leads to `/tips`

**Pull request merged**